jdk/src/solaris/bin/java_md_solinux.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1998, 2015, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1998, 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Don't modify the Oracle copyright, pls add an IBM copyright after the Oracle copyright and license.
```
/*
 * ===========================================================================
 * (c) Copyright IBM Corp. 2020, 2020 All Rights Reserved
 * ===========================================================================
 */
```